frontend/src/main/web/lib/dispatchers/GlossaryDispatcher.js

```diff
@@ -0,0 +1,19 @@
+import assign from 'object-assign';
+import {Dispatcher} from 'flux';
+
+var GlossaryDispatcher = assign({}, new Dispatcher(), Dispatcher.prototype, {
```

Comment:
What is this doing? It looks like it:

 - takes an empty object
 - constructs a Dispatcher and copies its properties into the new object
 - copies all the properties from the prototype of Dispatcher onto the new object (possibly overwriting some from the constructed Dispatcher)
 - copies a function prop onto the object, possibly overwriting the same property if it was present in the Dispatcher or its prototype.

It looks like it might be trying to do inheritance, but without actually doing any inheritance at all and just copying everything to one object instead.